Problem opening named pipes in Linux

I have a model that includes a manager script that communicates with another application (Simile) via a named pipe. This all worked when ApsimX was built on Mono. However, I just upgraded to version 6775 and I am getting a weird error message which I suspect may be something to do with the fact that it has moved to .Net Core 3.0. The message is:

APSIMef5d821f-8661-43cf-bf2d-f9d17764ea73.cs(45,24): error CS1069: The type name 'NamedPipeClientStream' could not be found in the namespace 'System.IO.Pipes'. This type has been forwarded to assembly 'System.IO.Pipes, Version=4.1.2.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' Consider adding a reference to that assembly.

I am executing Apsim using .NET Core SDK (3.1.412)

--- a/Models/Core/ScriptCompiler.cs
+++ b/Models/Core/ScriptCompiler.cs
@@ -334,6 +334,7 @@
                MetadataReference.CreateFromFile(typeof(System.ComponentModel.TypeConverter).Assembly.Location),
                MetadataReference.CreateFromFile(typeof(Console).Assembly.Location),
                MetadataReference.CreateFromFile(typeof(System.IO.File).Assembly.Location),
+               MetadataReference.CreateFromFile(typeof(System.IO.Pipes.PipeStream).Assembly.Location),
             };
 
             if (previousCompilations != null)

